### DISCUSSION TITLE
What's your best time for round table meeting?

### DISCUSSION THREAD

**Discussion by wuminzhe** [ringdao-community-guild/discussion#131] [General]
Now I have 3 roundtables scheduled in different time zones. But the attendance doesn't seem to be good. I'd like to know when is the best time for you to attend the meeting. Or can we find another way to hold the meeting?

Please leave your ideas in this discussion.

**Comment by SasoLithops** [comment]
I am sorry I didn't know there was a meeting.I would definitely attend. Got a few questions. Would be good if we knew a week or so in advance. I am fine Monday mornings between 7-9 a.m. UTC l. The rest of the days I mostly work on fluid schedule but can mostly arrange to jump into meeting if I know in week advance.

**Reply by wuminzhe** [reply]
yes, it's a good idea.

**Comment by 2xJDubs** [comment]
Hello Aki, I apologize as I didn't know these were recurring times.  Currently all 3 times do not work for me as well.  Here are my following availability:

Monday - after 9p UTC-6
Tuesday - after 930p UTC-6
Wednesday - 7a - 8a, 12p - 1p, after 9p UTC-6
Thursday - after 930p UTC-6
Friday - 7a - 8a, 12p - 1p, after 9p UTC-6

I hope we can find a times that work for everyone.

Thank you.

**Comment by hackfisher** [comment]
https://meet.felo.me/ 

Support live subtitles for multilingual meeting, good to be used in this meeting.

**Comment by wuminzhe** [comment]
After a study, `Friday 2pm UTC` may be a good choice. What do you think?
@PartnershipDude @SasoLithops @2xJDubs @nblogist and Lakiodsk, pleace check your or leave your prefer email address.

- [x] @PartnershipDude, partnershipdude@gmail.com
- [x] @2xJDubs, sherpa.elf@gmail.com
- [x] @SasoLithops, trilobitmanta@gmail.com
- [x] @nblogist, nblogist1@gmail.com
- [x] @Laki389, lazarosa@yahoo.com

**Reply by Laki389** [reply]
Hi @wuminzhe 
Laki - lazarosa@yahoo.com

p.s @Laki389 is my Github username :)

**Reply by 2xJDubs** [reply]
@wuminzhe i can't meet at that time.  After 2p UTC I have work meetings.

**Reply by SasoLithops** [reply]
Hi @wuminzhe !

I am fine with that time, but need to know at least three days in advance so I can arange being free at time.

My e-mail: trilobitmanta@gmail.com

Thank you.

**Comment by wuminzhe** [comment]
> Friday 2pm UTC works for me. I'll be at the meeting today.

> @wuminzhe i can't meet at that time. After 2p UTC I have work meetings.

@PartnershipDude @2xJDubs This proposed time is not for today. But for the meetings after today?

@2xJDubs I'll adjust the time. Can you give me a suggestion?

**Reply by Laki389** [reply]
@2xJDubs @wuminzhe 

Seeing Jesse post, Friday - 7a - 8a UTC-6, that is **13:00 - 14:00 UTC.**

@SasoLithops  wrote that he can adapt to the  time if he knows in advance.

That time is also ok for me.

@PartnershipDude Is that time okay for you too?

**Reply by 2xJDubs** [reply]
@wuminzhe Thank you for the clarification.  Like Laki mentioned I have some availability in my early morning.  Let me convert all of my times to UTC for clarity.

Monday - 300 - 430 UTC
Tuesday - 300 - 430 UTC
Wednesday - 330 - 500 UTC, 1300 - 1400 UTC, 1800 - 1900 UTC
Thursday - 300 - 430 UTC 
Friday - 330 - 500 UTC, 1300 - 1400 UTC, after 1800 UTC

**Comment by Laki389** [comment]
@wuminzhe @2xJDubs @PartnershipDude @SasoLithops 

**Friday(s) 13:00 UTC - 14:00 UTC.**

That is:

@wuminzhe 21:00 - 22:00 (if you are UTC+8)
@SasoLithops @PartnershipDude 15:00 - 16:00 for Saso, PD and me.
@2xJDubs 07:00 - 08:00 for Jesse.

A proposal that may suit everyone.

**Reply by wuminzhe** [reply]
Nice!

**Reply by JuneCribb** [reply]
Hello @wuminzhe, is the meeting open to all or the main team?

**Reply by SasoLithops** [reply]
@wuminzhe Aki what do you think? I am sure June being Darwinia advocate is welcome to join us.

**Reply by 2xJDubs** [reply]
@JuneCribb please leave your email.  As an Advocate I think you should be included.

**Reply by JuneCribb** [reply]
agbamihuoma@gmail.com can be used. 

Thank you!
